.github/workflows/runtime-extrinsics.yml

```diff
@@ -0,0 +1,69 @@
+name: Check Runtime n Extrinsics
+
+# This CI checks for changes in Creditcoin runtime and extrinsic ordering
+
+# Controls when the action will run.
+on:
+  pull_request:
+    branches: [dev]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  run_extrinsics:
+    name: Extrinsics
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node Dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - run: npm install --ignore-scripts -g @polkadot/metadata-cmp
```

Comment:
Which part of these scripts uses NodeJS ? I'm not clear why we install it here. 

Another point: In other yml files we use Node 16.x, maybe worth using the same version everywhere. (I will follow up with a PR to also use the same GitHub action)

Reply:
`line 68`
```
  changed_extrinsics=$(
    polkadot-js-metadata-cmp "$RELEASE_WS" "$HEAD_WS" \
      | sed 's/^ \+//g' | grep -e 'idx: [0-9]\+ -> [0-9]\+' || true
  )
```